README.md

```diff
@@ -1,6 +1,6 @@
 # Computer Vision Annotation Tool (CVAT)
 
-[![Build Status](https://travis-ci.org/openvinotoolkit/cvat.svg?branch=develop)](https://travis-ci.org/openvinotoolkit/cvat)
+[![CI](https://github.com/openvinotoolkit/cvat/workflows/CI/badge.svg?branch=develop&event=push)](https://github.com/openvinotoolkit/cvat/actions)
```

Comment:
Do we really need ``&event=push``?

Reply:
In this case, nightly will not be counted. Need to support nightly display too?

Reply:
I do not see any reasons to ignore nightly builds. Do you see?

Reply:
Thought to just separate nightly from the display. There are no reasons for this in general. Removed this filter.